src/_includes/layouts/ebook.njk

```diff
@@ -9,6 +9,14 @@ hubspot:
         <div class="md:flex md:flex-row gap-6">
             <div class="text-left md:w-1/2">
                 <h1>{{ contentTitle | safe }}</h1>
+                {% if secondaryImage and tertiaryImage %}
+                {% set imageSrc1 = ["./", secondaryImage] | join %}
+                {% set imageSrc2 = ["./", tertiaryImage] | join %}
+                <div class="my-6 flex gap-3 ebook-images">
+                    {% image imageSrc1, "FlowFuse Dashboard", [225] %}
+                    {% image imageSrc2, "FlowFuse Dashboard", [225] %}
```

Comment:
You're setting the same width for both images, but one is bigger than the other. The images aren't resized for the preview, so I'm not sure how this will look in production. Could you please update those numbers to match how they appear in the preview?